keycloak-extensions/src/test/java/org/alvearie/keycloak/KeycloakContainerTest.java

```diff
@@ -71,4 +81,28 @@ public void getHomePage() throws Exception {
         connection.setRequestMethod("GET");
         assertEquals(connection.getResponseCode(), 200);
     }
+    
+    @Test
+    public void passwordGrantLoginTest() {
+    	System.out.println("Host is: " + keycloak.getHost());
+    	String tokenURL = "http://localhost:" + keycloak.getHttpPort() + TOKEN_ENDPOINT;
+    	String jsonbody = "{\"grant_type\":\"password\",\"username\":\"" + USERNAME + "\",\"password\":\"" + PASSWORD + "\",\"client_id\":\""
+                + KC_CLIENT + "\"}";
+    	BasicHttpClient bh = new BasicHttpClient();
+        Map<String, Object> headers = new HashMap<String, Object>();
+        headers.put("Content-Type", "application/x-www-form-urlencoded");
+        Map<String, String> tokenDetails = new HashMap<>();
+        Response r;
+        try {
+            r = bh.execute(
+            		tokenURL,
+                    "POST", headers, null, jsonbody);
+
+            System.out.println(r.getEntity().toString());
+
+        } catch (Exception e) {
+            e.printStackTrace();
+           
+        }
```

Comment:
we probably don't want to catch this exception...that will prevent the test from failing.

Reply:
Done

Reply:
I'm still seeing the catch here, did you push your change yet?